boards/m5stack/m5stack_cores3/m5stack_cores3_esp32s3_procpu_se.yaml

```diff
@@ -0,0 +1,21 @@
+identifier: m5stack_cores3/esp32s3/procpu/se
+name: M5Stack CoreS3 SE PROCPU
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - dma
+  - i2c
+  - spi
+  - uart
+  - watchdog
+testing:
+  ignore_tags:
+    - bluetooth
+    - gpio
```

Comment:
Any reason to let `gpio` and `pinmux` in `ignore_tags`?

Reply:
No reason.
It just followed the existing m5stack_cores3_procpu.yaml.

The reason is that perhaps the gpio0 and gpio1 are not enabled.
Here, I want to focus on adding a variant, so I will submit the GPIO-enabling patch soon after it is merged and address this problem.

Reply:
I think it would be much better to amend the commit by moving this part and squash it with the last commit where you do the other changes.

Reply:
I am done squashing commits.
And according to this https://github.com/zephyrproject-rtos/zephyr/pull/78718#discussion_r1864274910
discussion, I removed the `ignore_tags:` section.